.github/workflows/ci.yml

```diff
@@ -52,9 +34,9 @@ jobs:
         fetch-depth: 1
         lfs: false
 
-    - name: Set default versions
-      id: set_default_versions
-      uses: ./source/.github/actions/default-versions
+    - name: Read JSON file
+      id: get_config
+      run: echo "config=$(jq -c . < ./source/.github/workflows/config.json)" >> $GITHUB_OUTPUT
```

Comment:
maybe its better to have a dedicated action read and provide individual var instead of using fromJson everywhere ?